app/controllers/queue_items_controller.rb

```diff
@@ -39,7 +39,7 @@ def is_already_in_queue?(video)
   end
 
   def update_list
-    ActiveRecord::Base.transaction do
+    ActiveRecord::Base.transaction(requires_new: true) do
```

Comment:
You can actually alter your database cleaner config to alleviate the need to do this (`require_new`). I believe the need is actually a bug (since this is not a nested transaction) that is fixed in the latest master branch if I remember right. I will comment on the `spec_helper` where the other change could be made so you are aware.